### PR TITLE
Removed the dependency on lombok version 1.16.20, which was used for the experimental versions of the @Builder and @Value annotations.

### PR DESCRIPTION
**Pull request by RohanTalip**
`@Builder` was promoted in lombok from experimental to the main package in v1.16.0 on January 26th, 2015, nearly 4 years ago.
`@Value` was promoted in lombok from experimental to the main package in v0.12.0 on July 16th, 2013, over 5 years ago.

The experimental versions of `@Builder` and `@Value` have been replaced with the main package versions, or otherwise removed where appropriate.

The chain and fluent options to `@Builder` are now the default, so the BuilderChainAndFluent test was removed.
The ValueExperimental test no longer seems relevant so it was also removed.
Other tests were updated as necessary.

### CODE REVIEW

**Comment by mplushnikov** [comment]
Thank you Rohan Talip! I will look on your PR next weekend!

File: src/main/java/de/plushnikov/intellij/plugin/processor/handler/BuilderHandler.java [hunk 1]
```diff
@@ -53,7 +53,7 @@
   private static final Collection<String> INVALID_ON_BUILDERS = Collections.unmodifiableSet(new HashSet<String>(Arrays.asList(
     Getter.class.getSimpleName(), Setter.class.getSimpleName(), Wither.class.getSimpleName(), ToString.class.getSimpleName(), EqualsAndHashCode.class.getSimpleName(),
     RequiredArgsConstructor.class.getSimpleName(), AllArgsConstructor.class.getSimpleName(), NoArgsConstructor.class.getSimpleName(),
-    Data.class.getSimpleName(), Value.class.getSimpleName(), lombok.experimental.Value.class.getSimpleName(), FieldDefaults.class.getSimpleName())));
+    Data.class.getSimpleName(), Value.class.getSimpleName(), FieldDefaults.class.getSimpleName())));
```
Comment (mplushnikov) [inline src/main/java/de/plushnikov/intellij/plugin/processor/handler/BuilderHandler.java]: In this class the methods:
```
  private boolean isFluentBuilder(@NotNull PsiAnnotation psiAnnotation)
  private boolean isChainBuilder(@NotNull PsiAnnotation psiAnnotation)
``` 
can be removed too.

Comment (mplushnikov) [reply src/main/java/de/plushnikov/intellij/plugin/processor/handler/BuilderHandler.java]: And you can for usages of this methods and simplify it accordingly

File: src/main/java/de/plushnikov/intellij/plugin/processor/clazz/ValueProcessor.java [hunk 2]
```diff
@@ -40,7 +40,7 @@
   @SuppressWarnings({"deprecation", "unchecked"})
```
[the remark this answers is not in the document]
Comment (RohanTalip) [reply src/main/java/de/plushnikov/intellij/plugin/processor/clazz/ValueProcessor.java]: Good point.  I made some changes in commit 2c3c1112fd530827bd8f4c23e57efc2d74258837.

File: src/main/java/de/plushnikov/intellij/plugin/processor/clazz/builder/BuilderClassProcessor.java [hunk 3]
```diff
@@ -26,7 +26,7 @@
 
   @SuppressWarnings({"deprecation", "unchecked"})
```
[the remark this answers is not in the document]
Comment (RohanTalip) [reply src/main/java/de/plushnikov/intellij/plugin/processor/clazz/builder/BuilderClassProcessor.java]: Also done in commit 2c3c1112fd530827bd8f4c23e57efc2d74258837.

File: src/main/java/de/plushnikov/intellij/plugin/processor/handler/BuilderHandler.java (hunk 1 above)
Comment (RohanTalip) [reply src/main/java/de/plushnikov/intellij/plugin/processor/handler/BuilderHandler.java]: Also done in commit 2c3c1112fd530827bd8f4c23e57efc2d74258837.

**Comment by RohanTalip** [comment]
I think I addressed your comments in commit 2c3c1112fd530827bd8f4c23e57efc2d74258837.

Please take another look.

**Comment by mplushnikov** [comment]
Thank you for your work @RohanTalip! Please also add your to list of contributors!

**Pull request merged**